src/main/java/ui/Ui.java

```diff
@@ -164,4 +164,24 @@ public void showChapterList(ArrayList<Chapter> chapters, int chapterCount) {
     public void showError(String error) {
         out.println(error);
     }
+
+    public void showModuleUnedited(Module module) {
+        out.println("The following module will be edited:");
+        out.println(module);
+    }
+
+    public void showModuleEdited(Module module) {
+        out.println("Edited module:");
+        out.println(module);
+    }
+
+    public void showChapterUnedited(Chapter chapter) {
+        out.println("The following chapter will be edited:");
+        out.println(chapter);
+    }
+
+    public void showChapterEdited(Chapter chapter) {
+        out.println("Edited chapter:");
+        out.println(chapter);
+    }
```

Comment:
Would it be better to have a general method for the chapter and module each, then pass the message to that method?